lib/puppet/provider/f5_sslcertificate/rest.rb

```diff
@@ -0,0 +1,41 @@
+require 'puppet/provider/f5'
+require 'json'
+
+Puppet::Type.type(:f5_sslcertificate).provide(:rest, parent: Puppet::Provider::F5) do
+
+  def create_message(basename, hash)
```

Comment:
`create_message` is a method from the parent provider that you're overriding.  But why?? It's not used anywhere.